libraries/AR_Motors/AP_MotorsUGV.cpp

```diff
@@ -126,6 +128,22 @@ const AP_Param::GroupInfo AP_MotorsUGV::var_info[] = {
     // @Increment: 0.1
     // @User: Standard
     AP_GROUPINFO("REV_DELAY", 15, AP_MotorsUGV, _reverse_delay, 0),
+
+    // @Param: BAT_CURR_MAX
```

Comment:
So I thought to myself, "we should probably move the maximum current to the battery monitor library as it's a property of the battery".

.... and (re)discovered `BATT_WATT_MAX`.

```
    // @Description: If battery wattage (voltage * current) exceeds this value then the system will reduce max throttle (THR_MAX, TKOFF_THR_MAX and THR_MIN for reverse thrust) to satisfy this limit. This helps limit high current to low C rated batteries regardless of battery voltage. The max throttle will slowly grow back to THR_MAX (or TKOFF_THR_MAX ) and THR_MIN if demanding the current max and under the watt max. Use 0 to disable.
```

Plane uses this code to limit current in the same way as you are in here.

Sorry, I should have picked this up earlier - we should use the common parameter.